instrumentation/opentelemetry/init.go

```diff
@@ -321,6 +382,42 @@ func RegisterServiceWithSpanProcessorWrapper(serviceName string, resourceAttribu
 	}), tp, nil
 }
 
+func initializeMetrics(cfg *config.AgentConfig) func() {
+	if cfg.GetTelemetry() == nil || !cfg.GetTelemetry().GetMetricsEnabled().GetValue() {
+		return func() {}
+	}
+
+	metricsExporterFactory := makeMetricsExporterFactory(cfg)
+	metricsExporter, err := metricsExporterFactory()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
This means if TPA is down instrumented app won't be able to start? Better to fail silently?

Reply:
If the metrics/traces collector is down and you bring up the go app with this agent in it, it comes up without issue. You do get context deadline messages since you cannot export anything. After you bring up the collector the spans and metrics start being sent.